disnake/interactions/application_command.py

```diff
@@ -174,16 +174,19 @@ class ApplicationCommandInteractionData(Dict[str, Any]):
         All resolved objects related to this interaction.
     options: List[:class:`ApplicationCommandInteractionDataOption`]
         A list of options from the API.
+    guild_id: Optional[:class:`int`]
+        ID of the guild the command is registered to.
     target_id: :class:`int`
-        ID of the user or message targetted by a user or message command
+        ID of the user or message targetted by a user or message command.
     target: Union[:class:`User`, :class:`Member`, :class:`Message`]
-        The user or message targetted by a user or message command
+        The user or message targetted by a user or message command.
```

Comment:
```suggestion
    target_id: :class:`int`
        ID of the user or message targeted by a user or message command.
    target: Union[:class:`User`, :class:`Member`, :class:`Message`]
        The user or message targeted by a user or message command.
```
technically unrelated typo fix, happened to notice this while reading through the code

Reply:
https://github.com/DisnakeDev/disnake/pull/1107/commits/c44806b42cbd6a815d2343cbc41313231c57a887